test/e2e/preflightCliTest_test.go

```diff
@@ -0,0 +1,128 @@
+package e2e
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestPreflightHelp(t *testing.T) {
+	got := preflightHelp()
+	want := "A utility that allows you to pre-test your bundles, operators, and container before submitting for Red Hat Certification.\n\nUsage:\n  preflight [command]\n\nAvailable Commands:\n  certify        Submits check results to Red Hat\n  check          Run checks for an operator or container\n  completion     generate the autocompletion script for the specified shell\n  help           Help about any command\n  runtime-assets Returns information about assets used at runtime.\n  support        Submits a support request\n\nFlags:\n  -h, --help      help for preflight\n  -v, --version   version for preflight\n\nUse \"preflight [command] --help\" for more information about a command.\n"
```

Comment:
These are going to be extremely fragile. I don't see the utility of this type of test. The majority of the help text is going to be auto-generated by cobra anyway. In fact, a change was made recently that already would have made these fail.

Reply:
Correct, this needs to be changed wrt changes in preflight help messages. 
I don't see the utility of this type of test--> This is just basic test if preflight is displaying correct help message

Reply:
Sure, I will remove the help tests